lib/vsc/utils/fancylogger.py

```diff
@@ -85,6 +85,15 @@
 import weakref
 from distutils.version import LooseVersion
 
+HAVE_COLOREDLOGS_MODULE = False
+if os.environ.get('FANCYLOGGER_NO_COLOREDLOGS', '0').lower() not in ('1', 'yes', 'true', 'y'):
```

Comment:
Aren't we using the `('1', 'yes', 'true', 'y')` in multiple places now?

In that case, this should become a constant like `ENABLE_VALUES`?